### ISSUE TITLE
🐞 Fail to export single node configuration

### ISSUE THREAD

**Issue by yanecc** [hiddify/hiddify-app#780]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

Exporting the Hyseria2 or Shadowsocks node configuration to the clipboard on Windows failed. It showed that the export was successful, but the content was actually blank.

### Minimal Reproducible Example (MRE)

1. Share the Hyseria2 or Shadowsocks node configuration.
2. Export the configuration to the clipboard.
3. Check the clipboard.

### Version

v1.1.1

### Platform/OS

Windows

**Comment by lymanjre** [comment]
Can you tell us if it can be imported successfully via subscription link?

**Comment by yanecc** [comment]
> Can you tell us if it can be imported successfully via subscription link?

Yes, but the Export feature is valid only for subscription links and is invalid for individual nodes.
